cpp/IceBox/hello/HelloServiceI.cpp

```diff
@@ -20,10 +22,10 @@ HelloServiceI::start(
     const shared_ptr<Ice::Communicator>& communicator,
     const Ice::StringSeq& /*args*/)
 {
-    _adapter = communicator->createObjectAdapter(name);
-    auto hello = make_shared<HelloI>();
-    _adapter->add(hello, Ice::stringToIdentity("hello"));
+    _adapter = communicator->createObjectAdapterWithEndpoints("Hello", "tcp -p 10000:udp -p 10000");
```

Comment:
(For a separate PR): we should stop using udp with random demos and showing off multi-endpoint OAs.